Riot/Modules/Contacts/Details/Views/RoomTableViewCell.m

```diff
@@ -66,16 +66,13 @@ - (void)render:(MXRoom *)room
     self.titleLabel.text = room.summary.displayname;
     
     self.directRoomBorderView.hidden = !room.isDirect;
-    
-    self.encryptedRoomIcon.hidden = !room.summary.isEncrypted;
```

Comment:
Does this work for newly created cells? Afaik `prepareForReuse` doesn't get called for newly initialized cells.

Reply:
yes because it is hidden in the xib.
I pushed a commit so that we do not touch it anymore from the code. This icon is now hidden and useless. I keep it because I have the feeling that this bottom right badge could be useful in the future.

Reply:
Ah ok, makes sense if hidden in xib. Thanks.